app/src/main/res/values-nb/strings.xml

```diff
@@ -0,0 +1,368 @@
+<resources>
+    <string name="wallet_activity_name">Lommebok</string>
+
+    <string name="menu_testnet">Testnett</string>
+    <string name="menu_about">Om</string>
+    <string name="menu_privacy">Personvernserklæring</string>
+
+    <string name="menu_share">Del</string>
+    <string name="menu_help">Hjelp</string>
+    <string name="menu_info">Detaljer</string>
+    <string name="menu_receive">Motta</string>
+    <string name="menu_rename">Gi nytt navn &#8230;</string>
+    <string name="menu_archive">Arkiver</string>
+    <string name="menu_backup">Backup</string>
+	<string name="menu_changepw">[Forandre passord]</string>
+
+    <string name="password_weak">Fortsett å skrive &#8230;</string>
+    <string name="password_fair">Meh &#8230;</string>
+    <string name="password_good">Kom igjen, du kan gjøre det bedre!</string>
+    <string name="password_strong">Nesten der &#8230;</string>
+    <string name="password_very_strong">Å ja, h4x0r stil!</string>
+
+    <string name="label_login_wallets">Lommebøker</string>
+    <string name="label_credits">Credits</string>
+    <string name="label_ok">OK</string>
+    <string name="label_cancel">Avbryt</string>
+    <string name="label_close">Lukk</string>
+    <string name="label_wallet_advanced_details">Trykk for detaljert informasjon</string>
+
+    <string name="label_send_success">Sendt suksessfullt!</string>
+    <string name="label_send_done">Ferdig</string>
+
+    <string name="label_receive_info_gen_qr_code">Trykk for QR kode</string>
+    <string name="info_send_prio_fees">Høyere prioritet = høyere avgifter</string>
+
+    <string name="info_xmrto_enabled">BTC betaling tilgjengelig, trykk for mer info.</string>
+
+    <string name="info_xmrto"><![CDATA[
+        <b>Du skrev inn en Bitcoin addresse.</b><br/>
+        <i>Du vil sende XMR og mottakeren vil få BTC gjennom <b>XMR.TO</b> tjenesten.</i>
+    ]]></string>
+
+    <string name="info_send_xmrto_success_order_label">XMR.TO Ordre</string>
+
+    <string name="info_send_xmrto_success_sending">Sender</string>
+    <string name="info_send_xmrto_success_status_label">Status:</string>
```

Comment:
pls delete this (info_send_xmrto_success_status_label) - does not exist in original